hazelcast-all/src/test/java/com/hazelcast/it/CheckAllDependenciesIT.java

```diff
@@ -6,6 +6,6 @@
 
     @Override
     protected boolean isMatching(String urlString) {
-        return urlString.contains("hazelcast-all-") && urlString.contains("target");
+        return urlString.contains("hazelcast/hazelcast/") && !urlString.contains("test");
```

Comment:
That looks tricky. What will be a matching url look like?

Reply:
Here is the matching URL:
`jar:file:/Users/hasancelik/.m2/repository/com/hazelcast/hazelcast/3.9-SNAPSHOT/hazelcast-3.9-SNAPSHOT.jar!/META-INF/MANIFEST.MF`
I have added `!urlString.contains("test")` part to avoid to match with `test` URL --> `jar:file:/Users/hasancelik/.m2/repository/com/hazelcast/hazelcast/3.9-SNAPSHOT/hazelcast-3.9-SNAPSHOT-tests.jar!/META-INF/MANIFEST.MF`